.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
     -   id: flake8
         types: [file, python]
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.782
+    rev: v0.942
```

Comment:
Please revert this change since it is not related to PR's goal of removing `pytest.warns`.